README.md

```diff
@@ -54,7 +54,8 @@ by their priority and status. It can be found [here](https://github.com/orgs/Rev
 Thrive is written in C# with a few helper scripts written in ruby. In
 order to work on the C# you need to compile Thrive yourself. You can
 find instructions for how to do that in the [setup
-instructions][setupguide].
+instructions][setupguide]. And if you've never used Godot before
+please read [learning Godot](LEARNING_GODOT.md).
```

Comment:
This link is still incorrect. The one below is fixed but this still needs to be changed. (you could maybe use the markdown syntax where the link is at the bottom of the file so that these two places linking to the same thing can't get out of sync.